api/src/main/java/io/strimzi/api/Annotations.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api;
+
+/**
+ * Class for holding some annotation keys and utility methods for handling
```

Comment:
There are no utility methods in this class, or?

Reply:
@scholzj - I have a suggestion instead of using the labels/annotations from new class in all the places. Can't we extend the `Labels` and `Annotations` class from `operator-common` module with the new class, the parent class(from `api` module) will be having all the labels which we moved and the child class(from `operator-common` module) can inherit them , this way I believe we can avoid doing the changes to all the places where it is used and duplicates will be handled as well. Let me know your point of view on this.

@ppatierno - You are right no utility methods, I will remove that from comment lines.